Makefile

```diff
@@ -12,9 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# STANDARD OPTIONS
+###############
+
 # Set DOCKER_REGISTRY to customise the image docker repo, e.g. "quay.io/jetstack"
 DOCKER_REGISTRY :=
+# Set APP_VERSION to customize the image tag, eg "v0.0.5-dev"
 APP_VERSION :=
+# Set the target platform to build for. Defaults to linux/amd64
+PLATFORM := @io_bazel_rules_go//go/toolchain:linux_amd64
+
+# OPTIONS YOU PROBABLY DON'T NEED TO MODIFY UNLESS DOING SOMETHING VERY SPECIFIC.
+###############
+
+# Set an alternative base image https://github.com/jetstack/cert-manager/blob/master/build/BUILD.bazel#L42
+BASE_IMAGE := static
+# Set to '' to not disable CGO. Disabled by default.
+CGO_DISABLED := --@io_bazel_rules_go//go/config:pure
```

Comment:
suggestion: I think the use of this flag is pretty surprising to me, as it's currently implemented.

I think what I'd expect would be that there'd be a boolean flag, `CGO_DISABLED`. We'd also have an additional variable, say `BAZEL_IMAGES_FLAGS`, and if `CGO_DISABLED` was set to `1` we'd append `--@io_bazel_rules_go//go/config:pure` to `BAZEL_IMAGES_FLAGS`.

I think this is closer to how I'd expect it, as a user. Something like this:

```make
CGO_DISABLED ?= 0
BAZEL_IMAGES_FLAGS ?=

ifeq ($(CGO_DISABLED),1)
	BAZEL_IMAGES_FLAGS += --@io_bazel_rules_go//go/config:pure
endif

.PHONY: images
images:
	bazel run $(BAZEL_IMAGES_FLAGS) ...
```

(it also gives us a new feature for free: people will be able to set `BAZEL_IMAGES_FLAGS` as an environment variable and pass arbitrary flags into `make images`, if they wanted to)

What do you think?

Reply:
Yeah agree that the name of the variable and the actual value doesn't help in understanding what it does. In this case the confusing bit is that it is the default case when we want to pass that flag not vice versa.

But 
```
CGO_ENABLED ?= 0
BAZEL_IMAGES_FLAGS ?=

ifeq ($(CGO_ENABLED),0)
	BAZEL_IMAGES_FLAGS += --@io_bazel_rules_go//go/config:pure
endif

.PHONY: images
images:
	bazel run $(BAZEL_IMAGES_FLAGS) ...
```
also looks better than what's there currently so might go for that and ability to pass arbitrary flags may be useful.

Reply:
I've updated the Makefile and tested that both targets work as expected with the default value of CGO_ENABLED.